[2025-06-03T20:29:45.170+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2025-06-03T20:29:43.535627+00:00 [queued]>
[2025-06-03T20:29:45.185+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2025-06-03T20:29:43.535627+00:00 [queued]>
[2025-06-03T20:29:45.186+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2025-06-03T20:29:45.214+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2025-06-03 20:29:43.535627+00:00
[2025-06-03T20:29:45.222+0000] {standard_task_runner.py:60} INFO - Started process 62870 to run task
[2025-06-03T20:29:45.227+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'create_postgres_table', 'manual__2025-06-03T20:29:43.535627+00:00', '--job-id', '366', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_dag.py', '--cfg-path', '/tmp/tmpo_jy53bv']
[2025-06-03T20:29:45.229+0000] {standard_task_runner.py:88} INFO - Job 366: Subtask create_postgres_table
[2025-06-03T20:29:45.315+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2025-06-03T20:29:43.535627+00:00 [running]> on host 4bea2aab752f
[2025-06-03T20:29:45.443+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-03T20:29:43.535627+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-03T20:29:43.535627+00:00'
[2025-06-03T20:29:45.446+0000] {sql.py:276} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2025-06-03T20:29:45.462+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-03T20:29:45.475+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-03T20:29:45.513+0000] {sql.py:450} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2025-06-03T20:29:45.545+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=create_postgres_table, execution_date=20250603T202943, start_date=20250603T202945, end_date=20250603T202945
[2025-06-03T20:29:45.604+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-03T20:29:45.646+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
